protocol/streaming/types/errors.go

```diff
@@ -0,0 +1,13 @@
+package types
+
+// DONTCOVER
```

Comment:
What is this?

Reply:
idk I just copy pasted from some other error.go 

probably auto generated previously by ignite or something. I can remove